.github/workflows/ci.yml

```diff
@@ -168,85 +168,43 @@ jobs:
-      - name: Install TYPO3 Core
-        env:
-          TYPO3: "${{ matrix.typo3-version }}"
-        run: |
-          composer require --no-ansi --no-interaction --no-progress --no-install typo3/cms-core:"$TYPO3"
-          composer show
-      - name: Install lowest dependencies with composer
-        if: "matrix.composer-dependencies == 'lowest'"
-        run: |
-          composer update --no-ansi --no-interaction --no-progress --with-dependencies --prefer-lowest
-          composer show
-      - name: Install highest dependencies with composer
-        if: "matrix.composer-dependencies == 'highest'"
+      - name: Install composer dependencies
         run: |
-          composer update --no-ansi --no-interaction --no-progress --with-dependencies
-          composer show
-      - name: Start MySQL
-        run: "sudo /etc/init.d/mysql start"
+          Build/Scripts/runTests.sh -p ${{ matrix.php-version }} -t ${{matrix.typo3-version}} -s composerUpdate${{matrix.composer-dependencies}}
       - name: Run functional tests
         run: |
-          export typo3DatabaseName="$DB_DATABASE";
-          export typo3DatabaseHost="$DB_HOST";
-          export typo3DatabaseUsername="$DB_USER";
-          export typo3DatabasePassword="$DB_PASSWORD";
-          composer ci:tests:functional
+          Build/Scripts/runTests.sh -p ${{ matrix.php-version }} -t ${{matrix.typo3-version}} -s functional
     strategy:
       fail-fast: false
       matrix:
         include:
-          - typo3-version: "^12.4"
+          - typo3-version: "12.4"
             php-version: "8.1"
-            composer-dependencies: lowest
-          - typo3-version: "^12.4"
+            composer-dependencies: Min
+          - typo3-version: "12.4"
             php-version: "8.1"
-            composer-dependencies: highest
-          - typo3-version: "^12.4"
+            composer-dependencies: Max
+          - typo3-version: "12.4"
             php-version: "8.2"
-            composer-dependencies: lowest
-          - typo3-version: "^12.4"
+            composer-dependencies: Min
+          - typo3-version: "12.4"
             php-version: "8.2"
-            composer-dependencies: highest
-          - typo3-version: "^12.4"
+            composer-dependencies: Max
+          - typo3-version: "12.4"
             php-version: "8.3"
-            composer-dependencies: lowest
-          - typo3-version: "^12.4"
+            composer-dependencies: Min
+          - typo3-version: "12.4"
             php-version: "8.3"
-            composer-dependencies: highest
-#          - typo3-version: "^12.4"
+            composer-dependencies: Max
+#          - typo3-version: "12.4"
 #            php-version: "8.4"
-#            composer-dependencies: lowest
-          - typo3-version: "^12.4"
+#            composer-dependencies: Min
+          - typo3-version: "12.4"
             php-version: "8.4"
-            composer-dependencies: highest
+            composer-dependencies: Max
```

Comment:
I propose we do this change (Min/Max and TYPO3 versions without `^` in the keys) across the board in a pre-patch to keep this PR here smaller.